web/profiles/sdd/config/install/block.block.seven_breadcrumbs.yml

```diff
@@ -1,19 +0,0 @@
-langcode: en
```

Comment:
You removed all of config/install folder, need to add them back with changes included in d9